Samples/auth/Outlook-Add-in-SSO-events/README.md

```diff
@@ -0,0 +1,103 @@
+
+## Applies to
+
+- Outlook on Windows
+- Outlook on the web
+- [new Outlook on Mac](https://support.microsoft.com/office/6283be54-e74d-434e-babb-b70cefc77439)
+
+## Prerequisites
+
+- A Microsoft 365 account. To get one, join the [Microsoft 365 Developer Program](https://aka.ms/devprogramsignup).
+- [Node.js](https://nodejs.org/) version 16 or greater.
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 8 or greater.
+
+To run the sample, you first need to create an app registration to use SSO.
+
+## Register the add-in with the Microsoft identity platform
+
+Before you run the sample, you must create an app registration to use SSO. Use the following values for the subsequent app registration steps.
+
+| Placeholder or section | Value          |
+|------------------------|----------------|
+| `<add-in-name>`          | `outlook-event-sso-sample` |
+| `<fully-qualified-domain-name>` | `localhost:3000` |
+| Microsoft Graph permissions | profile, openid, User.Read |
+
+Follow the steps in [Register an Office Add-in that uses single sign-on (SSO) with the Microsoft identity platform](https://learn.microsoft.com/office/dev/add-ins/develop/register-sso-add-in-aad-v2).
+
+> Note: The instructions tell you to create a redirect URI for a single-page application. This step isn't necessary for this sample because it doesn't use a fallback authentication approach if SSO fails.
+
+## Configure the sample
+
+1. Clone or download this repo.
+1. In Visual Studio Code (or editor of your choice), open the root folder for this sample.
+
+### Update manifest.xml
+
+1. Open the **manifest.xml** file.
+1. Find the `<WebApplicationInfo>` section near the bottom of the manifest. Then, replace the `Enter_client_ID_here` value, in both places where it appears, with the application ID you generated as part of the app registration process.
+
+    > Note: Make sure that the port number in the `Resource` element matches the port used by your project. It should also match the port you used when registering the application.
+
+1. Save your changes.
+
+### Update .ENV
+
+1. Open the **.ENV** file.
+1. Replace the `Enter_client_ID_here` placeholder value with the application ID you generated as part of the app registration process.
+1. Replace the `Enter_client_secret_here` placeholder value with the client secret you generated as part of the app registration process.
+1. Save your changes.
+
+## Run the sample
+
+1. Open a terminal window and run the command `npm install` to install all package dependencies.
+1. Run the command `npm start` to start the web server.
+1. To test the add-in in Outlook, you need to sideload it. Follow the instructions at [Sideload an Office Add-in for testing](https://learn.microsoft.com/office/dev/add-ins/outlook/sideload-outlook-add-ins-for-testing) to sideload the sample.
+1. Compose a new email. The email will display a notification that it will append a signature.
```

Comment:
This seems to be missing the "Sign in and consent" step.

Reply:
That step is done during registration, but I added a note here in case it gets missed. You can consent by opening the task pane, and choosing the consent button.